### DISCUSSION TITLE
Welcome to League Page Discussions!

### DISCUSSION THREAD

**Discussion by nmelhado** [nmelhado/league-page#116] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cuddebtj** [comment]
My league refuses to switch to Sleeper and I find this as a beautiful page. I know back end python and currently pull all my leagues data from the Yahoo API, Would you be willing to work together to set this up for Yahoo potentially? I'm familiar with JS, but not to the extent of manipulating this before the upcoming season.

**Reply by nmelhado** [reply]
Moved to https://github.com/nmelhado/league-page/discussions/145, in case others want to follow the discussion
